shanghai.example.yaml

```diff
@@ -4,6 +4,7 @@ networks:
     # serves as base for each network
     # networks can overwrite any setting here.
     nick: Shanghai
+    user: Shanghai
```

Comment:
No well.. the `user` is currently used for the [`USER`](https://tools.ietf.org/html/rfc2812#section-3.1.3) command upon connecting. In rfc1459 the username, specified for the `USER` command, had a [different meaning](https://tools.ietf.org/html/rfc1459#section-4.1.3), that's been removed in rfc2812 as it seems. There is also a section about [ident lookups](https://tools.ietf.org/html/rfc1459#section-8.11.2) in the old rfc. But this is pretty much obsolete nowadays. So I think it's fine to remove the ident argument.